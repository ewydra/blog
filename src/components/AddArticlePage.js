import React from "react";
import ArticleForm from "./ArticleForm";
import ErrorBoundary from "./utils/ErrorBoundary";

const AddArticlePage = () => (
  <ErrorBoundary><ArticleForm title="New article" /></ErrorBoundary>
);

export default AddArticlePage;
