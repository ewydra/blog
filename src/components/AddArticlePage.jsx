import React from "react";
import ArticleForm from "./ArticleForm";

const AddArticlePage = () => (
  <ArticleForm title="New article" />
);

export default AddArticlePage;
