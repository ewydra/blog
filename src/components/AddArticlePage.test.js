import React from 'react';
import { shallow } from 'enzyme';
import AddArticlePage from './AddArticlePage';

describe('AddArticlePage', () => {
  it('should render correctly', () => {
    const component = shallow(<AddArticlePage />);
    expect(component).toMatchSnapshot();
  });
});
