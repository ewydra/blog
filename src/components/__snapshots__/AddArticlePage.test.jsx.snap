// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddArticlePage should render correctly 1`] = `
<Connect(withRouter(ArticleForm))
  title="New article"
/>
`;
